### ISSUE TITLE
Doesn't work, yazi v0.4

### ISSUE THREAD

**Issue by c02y** [dawsers/toggle-view.yazi#2]

**Comment by dawsers** [comment]
Thanks. I will fix it as soon as I move to Yazi 0.4. My distribution is still using 0.3.3

**Comment by m-pauper** [comment]
Is there a time frame for this? I tried to fix it myself, changing the entry function to use the new `job` argument, and then access arg with `job.args[1]` but it still does not fix the problem.

```
Micro plugin failed:
runtime error: [string "toggle-view"]:9: attempt to index a nil value (global 'MANAGER')
stack traceback:
	[string "toggle-view"]:9: in function <[string "toggle-view"]:2>
```

**Comment by dawsers** [comment]
Sorry, Arch is still at 0.3.3

**Comment by dawsers** [comment]
Fixed in 9ae57f904d616f4462e829521ec1ad10727ec7ed
